components/gitpod-cli/cmd/timeout-set.go

```diff
@@ -49,7 +49,7 @@ For example, 30m, 1h, 2d, etc.`,
 			}
 			fail(err.Error())
 		}
-		fmt.Printf("Workspace timeout has been set to %d minutes.\n", int(duration.Minutes()))
+		fmt.Printf("Workspace timeout is set to %d minutes.\n", int(duration.Minutes()))
```

Comment:
Whilst it would be nice to have the same tense, I don't believe we have a good enough reason to update this. 

Can we revert? (the [same justification](https://github.com/gitpod-io/gitpod/pull/16091/files#r1090653124) as here)